installers/local/carbon.cpp

```diff
@@ -12,16 +8,19 @@ namespace path = llvm::sys::path;
 
 auto main(int argc, char** argv) -> int {
   llvm::StringRef bin = path::filename(argv[0]);
+
   if (bin == "carbon-explorer") {
-    static int static_for_main_addr;
+    static int static_for_main_address;
+
     std::string exe = fs::getMainExecutable(
-        argv[0], static_cast<void*>(&static_for_main_addr));
+            argv[0], static_cast<void *>(&static_for_main_address));
     llvm::StringRef install_path = path::parent_path(exe);
     llvm::SmallString<256> prelude_file(install_path);
     path::append(prelude_file, "data", "prelude.carbon");
+    
     return Carbon::ExplorerMain(prelude_file, argc, argv);
   } else {
-    fprintf(stderr, "Unrecognized Carbon binary requested: %s", argv[0]);
-    return 1;
+    return fprintf(stderr, "Unrecognized Carbon binary requested: %s", argv[0]),
+           1;
```

Comment:
I'm not sure use of comma operator expression makes things easier to understand here :thinking:

Reply:
Would you explain me why?